openshift_tools/monitoring/hawk_client.py

```diff
@@ -76,7 +76,11 @@ def push_metrics(self, unique_metric_list):
             _id = metric.host
             key = '{0}/{1}/{2}'.format(_type, _id, metric.key)
 
-            if isinstance(value, basestring):
+            if "heartbeat" in metric.key:
```

Comment:
Don't catch / translate the ops-runner metrics.

I think we want to clamp this down to just heartbeat.ping.